routers/comments.py

```
@@ -0,0 +1,60 @@
+@router.patch("/{commentId}")
+async def update_comment(commentId: int, comment: CommentUpdate):
+    """댓글 수정"""
+    # TODO: 실제 댓글 수정 로직 구현
+    return {
+        "status": "success",
+        "message": "댓글이 수정되었습니다.",
+        "data": {}
+    }
+
+@router.delete("/{commentId}", status_code=204)
+async def delete_comment(commentId: int):
```

Ruff (0.14.11)

Unused function argument: `commentId`

(ARG001)